Inclued Folders' Contents in the Zipped Archive

Ga68 opened the pull request.

Ga68 committed:
Update zip.py

Adding folder functionality

Ga68 committed:
Update README.md

nikipore reviewed (changes requested):
Now that you request a pull, you should no longer refer to your amendment as a "fork".

And your solution doesn't properly recurse into subdirectories. [This one](https://stackoverflow.com/a/25650295) should work. Please test and amend.

Ga68 committed:
Update README.md

Commit:
Add folder ability; create an Alfred set-able default file name and location to use

Ga68 commented:
I fixed up the README as you suggested.

I looked at the link you provided for the zipping-a-directory and the limitation of that method is that it will automatically create a new archive. I did however, use your suggestion to strength the folder-traversal to using [os.walk](https://docs.python.org/3.5/library/os.html#os.walk). In the process I set it up so that given the initial file list you send to the utility, all the files and folders get added to the "root directory" of the archive (regardless of where they come from on your computer...ex. across multiple folders), but then all sub-files/directories of directories initially passed will maintain their directory structure as originally made.

I added the capability to accept an Alfred environment variables for a default Archive location (ex. "I always want it to show up on my Desktop") and for a default Archive name (ex. "I want my zipped files to be called Zippy.zip by default").
